### PR TITLE
SIMD i16x8/i32x4/i64x2 shl implementation.

### PR DESCRIPTION
**Pull request by lizhengxing**

### CODE REVIEW

**Comment by lizhengxing** [comment]
@binji 
SIMD i16x8/i32x4/i64x2 shl implementation. PTAL!
Thanks!

@arunetm FYI!

File: test/interp/simd-shift.txt [hunk 1]
```diff
@@ -10,8 +10,44 @@
...
+    i32.const 3
+    i16x8.shl)
+  (func (export "i16x8_shl_1") (result v128)
+    v128.const i32 0xff000071 0xe0000702 0x00000003 0x00000004
+    i32.const 19
+    i16x8.shl)
+
+  ;; i32x4 shl
+  (func (export "i32x4_shl_0") (result v128)
+    v128.const i32 0xff000071 0xe0000702 0x00000003 0x00000004
```
Comment (binji) [inline test/interp/simd-shift.txt]: please modify this test to show bits shifting in the full 32-bit range -- this output is identical to the i16x8 shift so it is hard to see that it is working.

File: test/interp/simd-shift.txt [hunk 2]
```diff
@@ -10,8 +10,44 @@
...
+    i32.const 3
+    i32x4.shl)
+  (func (export "i32x4_shl_1") (result v128)
+    v128.const i32 0xff000071 0xe0000702 0x00000003 0x00000004
+    i32.const 35
+    i32x4.shl)
+
+  ;; i64x2 shl
+  (func (export "i64x2_shl_0") (result v128)
+    v128.const i32 0xff000071 0xe0000702 0x00000003 0x00000004
```
Comment (binji) [inline test/interp/simd-shift.txt]: Same here, please use full 64-bit range.

File: test/interp/simd-shift.txt (hunk 1 above)
Comment (lizhengxing) [reply test/interp/simd-shift.txt]: Done.

File: test/interp/simd-shift.txt (hunk 2 above)
Comment (lizhengxing) [reply test/interp/simd-shift.txt]: Done.

**Comment by lizhengxing** [comment]
@binji 
Updated the tests, PTAL!
Thanks!

@arunetm FYI!

**Pull request merged**